websites/R/RTL+/presence.ts

```diff
@@ -0,0 +1,154 @@
+const presence = new Presence({
+		clientId: "1033504954763198545",
+	}),
+	strings = presence.getStrings({
+		play: "general.playing",
+		pause: "general.paused",
+	}),
```

Comment:
Better way of getting strings.
```suggestion
	}),
```